guide/coaching.md

```diff
@@ -8,24 +8,28 @@ permalink: /guide/coaching/
 ---
 <h1>Becoming a Coach</h1>
 
-<p>You heard about Rails Girls Summer of Code and want to be a coach for a team in your city? Here you’ll find all you need to know about what coaching actually means.</p>
+<p>You heard about Rails Girls Summer of Code and want to be a coach for a team? Here you’ll find all you need to know about what coaching actually means.</p>
 
 <ul>
 <li><a href="#intro">What it takes to be a coach</a></li>
 <li><a href="#why">Why should I be a coach?</a></li>
-<li><a href="#what">What does coaching require?</a></li>
+<li><a href="#remote">Can I coach remotely?</a></li>
+<li><a href="#what">What will I be doing as a coach?</a></li>
+<li><a href="#expertise">Should I have a good knowledge of a project chosen by my team?</a></li>
```

Comment:
change wording here as well (to same as below): Should I have in-depth knowledge of the project(s) chosen by my team?